hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestJdbcSource.java

```diff
@@ -438,7 +438,7 @@ public void testSourceWithStorageLevel() {
   private void writeSecretToFs() throws IOException {
     FileSystem fs = FileSystem.get(new Configuration());
     FSDataOutputStream outputStream = fs.create(new Path("file:///tmp/hudi/config/secret"));
-    outputStream.writeBytes("jdbc");
+    outputStream.writeBytes("");
```

Comment:
Similar here for the password.  And should we add the password back for testing the secret (this refactoring of static variables and password should be in a separate PR)?

Reply:
reverted, will do it in separate PR.